pyvrp/cpp/Solution.h

```diff
@@ -95,27 +98,61 @@ class Solution
         [[nodiscard]] Load excessLoad() const;
 
         /**
-         * Total route duration, including waiting time.
+         * Total route duration, including travel, service and waiting time.
          */
         [[nodiscard]] Duration duration() const;
 
         /**
-         * Total duration of service on the route.
+         * Total duration of service on this route.
          */
         [[nodiscard]] Duration serviceDuration() const;
 
         /**
-         * Amount of time warp incurred along the route.
+         * Amount of time warp incurred on this route.
          */
         [[nodiscard]] Duration timeWarp() const;
 
+        /**
+         * Total duration of travel on this route.
+         */
+        [[nodiscard]] Duration travelDuration() const;
+
         /**
          * Total waiting duration on this route.
          */
         [[nodiscard]] Duration waitDuration() const;
 
         /**
-         * Release time of visits on this route.
+         * Start time of this route. This is the earliest possible time at which
+         * the route can leave the depot and have a minimal duration. It may be
+         * possible to leave earlier, but that would incur additional waiting
+         * time.
+         *
+         * .. note::
+         *
+         *    It may also be possible to leave later, without (additional)
+         *    violations of time window constraints or increasing the route
+         *    duration. In that case there is slack in the schedule: the amount
+         *    of slack can be obtained through :meth:`~slack`.
```

Comment:
I'm fine with this but I think it makes more sense if the description mentions the standard/preferred situation (leaving between earliest and latest start) and the note mentions the exceptions (leaving before earliest and after latest). Something along the lines of (did not check formatting).

```suggestion
         * Start time of this route. This is the earliest possible time at which
         * the route can leave the depot and have a minimal duration and time warp.
         * If :meth:`~slack` > 0, the start time can be delayed by at most the slack
         * without increasing the total (minimal) duration or time warp.
         *
         * .. note::
         *
         *    It may be possible to leave before the start time (if the depot time window
         *    allows for it) but this will introduce additional waiting time, increasing the
         *    total duration which will then no longer be minimal.
         *    Leaving after start time + slack will always increase the time warp, which, if
         *    it the time warp is currently 0, will cause the route to become infeasible w.r.t.
         *    time windows.
```